core/dbt/compilation.py

```diff
@@ -250,9 +250,6 @@ def _inject_ctes_into_sql(self, sql: str, ctes: List[InjectedCTE]) -> str:
 
         return str(parsed)
 
-    def _get_dbt_test_name(self) -> str:
```

Comment:
Wee, since we don't prepend tests with `count(*)` when "compiling" this can go away!